src/hoc/selectTable/index.js

```diff
@@ -31,8 +32,8 @@ export default (Component, options) => {
         checked,
         onClick: toggleSelection,
         selectType,
-        id: row[keyField],
         row,
+        id: `select-${row[keyField]}`
```

Comment:
Would this change make more sense on the defaultSelectInputComponent? If the intent is solely to create a unique ID for automation purposes, it seems wrong to break the pattern for managing the selection state in the SelectTable. 

If this is the approach that'll be used moving forward, I think we need an additional pull request to clarify what is being sent back, and update the example for SelectTable to maybe pull the selection ID from the row argument. 

I'm happy to create either, but would just ask for some direction from the powers at be as to which approach they would prefer; maybe I am missing something obvious and/or just flat out wrong here. :-)

Thanks in advance!

Reply:
For example:
`const defaultSelectInputComponent = props => {
  return (
    <input
      type={props.selectType || 'checkbox'}
      aria-label={`${props.checked ? 'Un-select':'Select'} row with id:${props.id}` }
      checked={props.checked}
      id={`select-${props.id}`}
      onClick={e => {
        const { shiftKey } = e
        e.stopPropagation()
        props.onClick(props.id, shiftKey, props.row)
      }}
      onChange={() => {}}
    />
  )
}`

Reply:
Can someone comment on this? This broke for me too. I now need to reference row.id instead of id for the proper keyField id.

Reply:
@mikemanyoung
The main thread is here #1198
My suggestion is to stay 6.8.6 fixed, and wait for next major release.